cartoframes/datasets.py

```diff
@@ -91,7 +97,7 @@ def _cartodbfy_query(self):
     def _copyfrom(self, with_lonlat=None):
         geom_col = _get_geom_col_name(self.df)
 
-        columns = ','.join(_norm_colname(c) for c in self.df.columns if c not in Dataset.RESERVED_COLUMN_NAMES)
+        columns = ','.join(norm for norm, orig in _normalize_column_names(self.df))
```

Comment:
Upss, you're right. I've moved the normalization method to the constructor, since we only allow passing the `dataframe` when creating a new `Dataset` a think it makes sense doing the column names processing there.